### PR TITLE
WIP: Conda env for windows devs

### PR DESCRIPTION
**Pull request by strseb**
## Description
Same as we did for macos, but windows. 
The status is currently: "works for me", we need to update Tc and check in what to do about the msvc redist. 

# Todo: 
- [ ] Update taskcluster to use this toolchain.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -11,10 +11,6 @@
 	path = 3rdparty/wireguard-apple
 	url = https://github.com/mozilla/wireguard-apple
         ignore=all
-[submodule "3rdparty/openSSL"]
```
Comment (strseb) [inline .gitmodules:14]: ooops: unrelated change. But it's actually valid to remove.

**Comment by birdsarah** [comment]
Again will pick this up in the doc.

Per decision in Feb retro, closing as more than 2 weeks without activity. You can re-open whenever.
